app/helpers/proposals_helper.rb

```diff
@@ -11,4 +11,13 @@ def datespan_header(start_date, end_date)
       end
     end
   end
+
+  def truncate_link(link)
+    if (link.size > 70)
+      truncated_link = link[0...69] << "..."
+      return truncated_link
+    else
+      return link
+    end
+  end
```

Comment:
Thanks for the advice! I will amend my code to reflect that. Slightly embarrassed : X